redhat-developer/app-services-cli
# WIP: feat(login): prompt user to skip mas sso login

### Description

Allow user to skip mas-sso auth through interactive mode.

### Verification Steps

1. Login to RHOAS CLI.
> go run ./cmd/rhoas login
2. After logging in to RH SSO, it should display: 
```
Logging in to MAS-SSO (Press any key to cancel) ...
```
3. Any key press should skip the MAS-SSO login and display
```
You are now logged in as "<username>"
```

### Type of change



### Checklist
